components/ipfs/ipfs_service.cc

```diff
@@ -661,10 +675,16 @@ void IpfsService::ImportTextToIpfs(const std::string& text,
 void IpfsService::OnImportFinished(ipfs::ImportCompletedCallback callback,
                                    size_t key,
                                    const ipfs::ImportedData& data) {
+  bool is_import_success{data.state == ipfs::IPFS_IMPORT_SUCCESS};
+
   if (callback)
     std::move(callback).Run(data);
 
   importers_.erase(key);
+
+  if (is_import_success && ipfs_service_delegate_) {
+    ipfs_service_delegate_->ShowAlwaysStartInfoBar(this);
```

Comment:
IpfsService doesn't know anything about infobars, so this method should be called OnImportToIpfsFinished

Reply:
done